src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/SentenceResponse.java

```diff
@@ -11,17 +11,25 @@
 import lombok.NoArgsConstructor;
 import uk.gov.justice.probation.courtcaseservice.service.model.UnpaidWork;
 
-@ApiModel(description = "Conviction Response")
+@ApiModel(description = "Sentence Response")
 @Data
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
 @JsonInclude(Include.NON_NULL)
-public class ConvictionResponse {
+public class SentenceResponse {
```

Comment:
See previous comment about immutability, although I think we are going to run into merge issues :-)